config/environments/development.rb

```diff
@@ -47,4 +47,5 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   config.hosts << "frontend.dev.gov.uk"
+  config.hosts << "example.org" # for pact tests
```

Comment:
I'm wondering where this comes from? Is it because Pact is making requests to the server with the `Host` set to `example.org`? Could we change it to use `frontend.dev.gov.uk`?

Reply:
Yes - it failed without this line, and I remember it being added to Collections. But on a second look, it looks like [it's been removed](https://github.com/alphagov/collections/commit/0a0523fd42ab1787716999938029e23ff7e4bcb7), so I'll apply the same technique here and explain it in a commit.

Reply:
I'm a bit late to this, but if you do `config.hosts.clear` you don't need to specify them all.